features/FxA-89-devices-view/README.md

```diff
@@ -356,21 +354,21 @@ according to the latest designs.
 
 Acceptance criteria:
 
-* [ ] When I connect a single desktop Firefox to Sync,
+* [x] When I connect a single desktop Firefox to Sync,
       my device list shows that device
       along with placeholders for iOS and Android,
       each with a "Get app" button
       linking to the relevant app store.
-* [ ] When viewing my device list
+* [ ] (Moved to Phase 3) When viewing my device list
```

Comment:
The "moved to Phase 3" here doesn't display in rendered view, I think it tries to make a link or something